addons/knobs/src/components/types/Array.js

```diff
@@ -47,6 +47,10 @@ ArrayType.propTypes = {
 };
 
 ArrayType.serialize = value => value;
-ArrayType.deserialize = value => value;
+ArrayType.deserialize = value => {
+  if (Array.isArray(value)) return value;
+
+  return Object.keys(value).reduce((array, key) => [...array, value[key]], []);
```

Comment:
Does this rely on `Object.keys` being sorted? It's actually not guaranteed to be:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys
> The Object.keys() method returns an array of a given object's own enumerable properties, in the same order as that provided by a for...in loop

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in#Array_iteration_and_for...in
> There is no guarantee that for...in will return the indexes in any particular order

Please sort the keys before reducing

Reply:
@Hypnosphi as evidenced in the unit test, the order of keys does not matter.

Reply:
@Hypnosphi 2f22c2c